openapi.yaml

```diff
@@ -12273,13 +12273,14 @@ components:
           x-linode-cli-display: 4
         object_storage:
           type: string
+          readOnly: true
```

Comment:
@Dorthu I added readOnly to this--is that accurate? when I read the code it didn't seem like it would accept changes to this value in PUT /account/settings

Reply:
This is accurate - you enable object storage by using it, and you disable it by POSTing to object-storage/cancel